packages/xod-client/src/editor/codemirrorXodMode.js

```diff
@@ -3,19 +3,65 @@ import 'codemirror/addon/mode/simple';
 import 'codemirror/addon/mode/overlay';
 import 'codemirror/mode/clike/clike';
 
+import 'codemirror/addon/edit/closebrackets';
+import 'codemirror/addon/edit/trailingspace';
+import 'codemirror/addon/comment/comment';
+import 'codemirror/addon/scroll/simplescrollbars';
+
 import { getTokens } from 'xod-arduino';
 
 (() => {
   let xodOverlay;
   CodeMirror.defineMode('xodCpp', (config, parserConfig) => {
     xodOverlay = CodeMirror.simpleMode(config, {
       mode: { spec: CodeMirror.modes.clike },
-      start: getTokens(),
+      start: getTokens().concat(
+        [
+          { regex: /\/\/.*/, token: 'comment' },
+          { regex: /\/\*/, token: 'comment', next: 'comment' },
```

Comment:
Quite hard to understand why `getTokens` are external and the comments are concated here. I’m pretty sure the `getTokens` should not belong to `xod-arduino` at all (see its `README`, the better name for the package would be `xod-arduino-transpiler`).

Wouldn’t it more clear to get rid of `getTokens` in a separate module and define them right in this module as they are much more codemirror-related than Arduino-related? There automatically will be less hassle with the config split.